### PR TITLE
splits qs configuration from application.configuration

### LINKED ISSUE #363: Separate quickstarter config from application.properties
**Is your feature request related to a problem? Please describe.**
At the moment, the `ConfigMap application.properties` defines which quickstarters are available: https://github.com/opendevstack/ods-provisioning-app/blob/master/ocp-config/prov-app/cm.yml. This is not ideal as it does not allow to compare the content of the file easily via Tailor when users change which quickstarters they want to use (e.g. if they point to internal quickstarters).

**Describe the solution you'd like**
It would be good to have the quickstarter configuration separately. This could be another ConfigMap, or a file. A ConfigMap has the advantage to be easily modifiable in the OCP UI.

### PR DESCRIPTION
**Pull request by stitakis**
fixes #363

### CODE REVIEW

File: openshift/cm.yml [hunk 1]
```diff
@@ -293,3 +245,67 @@ objects:
       spring.mail.password=${PROV_APP_MAIL_PASSWORD}
       provison.mail.sender=provision@${PROV_APP_MAIL_HOST}
       spring.main.allow-bean-definition-overriding=true
+
+- kind: ConfigMap
+  metadata:
+    labels:
+      app: prov-app
+    name: quickstarters.properties
```
Comment (michaelsauter) [inline openshift/cm.yml:253]: You'd then need to add `preserve bc:quickstarters.properties:/data` to the `Tailorfile` so no drift is generated for this.

**Comment by oalyman** [comment]
@michaelsauter @stitakis LGTM

**Comment by michaelsauter** [comment]
@stitakis Should we move forward with this? The feature is quite crucial and I don't see anything that would block your change?

**Comment by stitakis** [comment]
@michaelsauter we can move forward, do you approve this change?

**Comment by stitakis** [comment]
closing this in favour of #458

**Pull request closed without merging**